armi/operators/operatorMPI.py

```diff
@@ -163,6 +163,8 @@ def workerOperate(self):
                 note = context.MPI_COMM.bcast("wait", root=0)
                 if note != "wait":
                     raise RuntimeError('did not get "wait". Got {0}'.format(note))
+            elif cmd == "reset":
```

Comment:
I actually think ARMI has _way_ too many log statements as it is.  People add them, forget they exist, and never look at them again. Happens all the time.

Why is this one necessary for people to see during ever run?

Reply:
Definitely helped with confirming that this implementation was working. This specific one is only outputted on worker nodes, but there is one in `_finalizeInteract` that is logged on root. I can get rid of both, one of them, or none of them. I'll leave that to your decision.

Reply:
Indeed. I definitely want to avoid what's in the `else` block with this new command.